pkg/virt-handler/vm_test.go

```diff
@@ -125,6 +126,10 @@ var _ = Describe("VirtualMachineInstance", func() {
 
 	var certDir string
 
+	getCgroupManager = func(_ *v1.VirtualMachineInstance) (cgroup.Manager, error) {
+		return nil, nil
+	}
```

Comment:
The part with passing cgroup manager as an argument to hotplug volume mounter is covered I think by adjusted mounter mock. Apart from that I am not sure the tests here are supposed to cover cgroup functionality (previously cgroup part was hidden within the hotplug volume mounter and thus not called here at all because of the mock).